content/references/protocol-reference/ledger-data/ledger-object-ids.md

```diff
@@ -6,6 +6,7 @@ labels:
   - Data Retention
 ---
 # Ledger Object IDs
+[[Source]](https://github.com/ripple/rippled/blob/master/src/ripple/protocol/impl/Indexes.cpp)
```

Comment:
Would it be better to link to the spot where the IDs are defined? - https://github.com/ripple/rippled/blob/d49b4862244d9359c046bd04d48237591516cb6a/src/ripple/protocol/impl/Indexes.cpp#L46

Reply:
Although I guess that link could become outdated if rippled updates, nevermind!